tests/serverpod_test_server/test_integration/logging/endpoint_method_stream_logging_test.dart

```diff
@@ -0,0 +1,375 @@
+import 'dart:async';
+
+import 'package:serverpod/serverpod.dart';
+import 'package:serverpod_test_client/serverpod_test_client.dart';
+import 'package:serverpod_test_server/test_util/builders/log_settings_builder.dart';
+import 'package:serverpod_test_server/test_util/builders/runtime_settings_builder.dart';
+import 'package:serverpod_test_server/test_util/logging_utils.dart';
+import 'package:serverpod_test_server/test_util/test_serverpod.dart';
+import 'package:test/test.dart';
+
+Future<T> awaitFirst<T>(Stream<T> stream) async {
+  var completer = Completer<T>();
+  stream.listen((event) {
+    completer.complete(event);
+  });
+  return completer.future;
+}
+
+void main() async {
+  var client = Client('http://localhost:8080/');
+  late Serverpod server;
+  late Session session;
+
+  group('Database logger -', () {
```

Comment:
Suggestion: Replace this group `Given that continous logging is turned on` and `Given that continuous logging is turned off`.

The mix of scenarios makes it hard to understand if all scenarios are covered. A better grouping could probably help a bit. Or maybe I don't just understand the test order pattern.